[2023-05-17T16:46:19.189+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T16:46:19.220+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T16:46:19.222+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T16:46:19.256+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-17T16:46:19.268+0000] {standard_task_runner.py:57} INFO - Started process 1211 to run task
[2023-05-17T16:46:19.272+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmp1xw01hzk']
[2023-05-17T16:46:19.275+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask clean_fondue_data
[2023-05-17T16:46:19.355+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 8cf8fb9e28f7
[2023-05-17T16:46:19.565+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-17T16:46:19.771+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T16:46:19.851+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T16:46:19.870+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230517T164619, end_date=20230517T164619
[2023-05-17T16:46:20.047+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T16:46:20.102+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
<<<<<<< HEAD
[2023-05-17T17:55:01.092+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T17:55:01.112+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T17:55:01.114+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T17:55:01.138+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-17T17:55:01.150+0000] {standard_task_runner.py:57} INFO - Started process 702 to run task
[2023-05-17T17:55:01.154+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpk2bc_dgp']
[2023-05-17T17:55:01.157+0000] {standard_task_runner.py:85} INFO - Job 24: Subtask clean_fondue_data
[2023-05-17T17:55:01.234+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host a37c7bc3731c
[2023-05-17T17:55:01.371+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-17T17:55:01.484+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T17:55:01.541+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T17:55:01.557+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230517T175501, end_date=20230517T175501
[2023-05-17T17:55:01.608+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T17:55:01.721+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:18:11.260+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T18:18:11.299+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T18:18:11.300+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:18:11.555+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-17T18:18:11.570+0000] {standard_task_runner.py:57} INFO - Started process 845 to run task
[2023-05-17T18:18:11.591+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmphftzgus3']
[2023-05-17T18:18:11.598+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask clean_fondue_data
[2023-05-17T18:18:12.167+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 3390ff13fbf6
[2023-05-17T18:18:12.665+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-17T18:18:12.750+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:18:13.158+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:18:13.173+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230517T181811, end_date=20230517T181813
[2023-05-17T18:18:13.880+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:18:13.922+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:25:49.098+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T18:25:49.141+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T18:25:49.141+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:25:49.181+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-17T18:25:49.190+0000] {standard_task_runner.py:57} INFO - Started process 963 to run task
[2023-05-17T18:25:49.203+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpdhwboe9j']
[2023-05-17T18:25:49.205+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask clean_fondue_data
[2023-05-17T18:25:49.282+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 70fcd41af239
[2023-05-17T18:25:49.441+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-17T18:25:49.612+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:25:49.715+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:25:49.750+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230517T182549, end_date=20230517T182549
[2023-05-17T18:25:49.826+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:25:49.873+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:37:02.774+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T18:37:02.790+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-17T18:37:02.791+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:37:02.813+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-17T18:37:02.822+0000] {standard_task_runner.py:57} INFO - Started process 656 to run task
[2023-05-17T18:37:02.825+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpetcxxtx5']
[2023-05-17T18:37:02.827+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask clean_fondue_data
[2023-05-17T18:37:02.896+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 1545bed90dab
[2023-05-17T18:37:03.040+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-17T18:37:03.175+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:37:03.227+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:37:03.243+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230517T183702, end_date=20230517T183703
[2023-05-17T18:37:03.279+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:37:03.474+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-18T06:41:12.710+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T06:41:12.748+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T06:41:12.749+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T06:41:12.782+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-18T06:41:12.792+0000] {standard_task_runner.py:57} INFO - Started process 1204 to run task
[2023-05-18T06:41:12.798+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmp88vt0243']
[2023-05-18T06:41:12.801+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask clean_fondue_data
[2023-05-18T06:41:12.888+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 714a0c7ad307
[2023-05-18T06:41:13.180+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-18T06:41:13.308+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T06:41:13.374+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T06:41:13.393+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230518T064112, end_date=20230518T064113
[2023-05-18T06:41:13.452+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T06:41:13.726+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-18T06:53:30.629+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T06:53:30.649+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T06:53:30.649+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T06:53:30.700+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-18T06:53:30.710+0000] {standard_task_runner.py:57} INFO - Started process 1568 to run task
[2023-05-18T06:53:30.723+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpjt68jnil']
[2023-05-18T06:53:30.727+0000] {standard_task_runner.py:85} INFO - Job 41: Subtask clean_fondue_data
[2023-05-18T06:53:30.854+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 26547302378e
[2023-05-18T06:53:31.081+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-18T06:53:31.328+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T06:53:31.448+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T06:53:31.477+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230518T065330, end_date=20230518T065331
[2023-05-18T06:53:31.532+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T06:53:31.579+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
=======
[2023-05-18T04:22:15.753+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T04:22:15.775+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T04:22:15.776+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T04:22:15.841+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-18T04:22:15.856+0000] {standard_task_runner.py:57} INFO - Started process 473 to run task
[2023-05-18T04:22:15.860+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpeew3ejuv']
[2023-05-18T04:22:15.862+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask clean_fondue_data
[2023-05-18T04:22:16.014+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host b36de7cfc389
[2023-05-18T04:22:16.288+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-18T04:22:16.441+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T04:22:16.518+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T04:22:16.538+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230518T042215, end_date=20230518T042216
[2023-05-18T04:22:16.736+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T04:22:16.783+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
>>>>>>> 8e4a41b368593e9ce283ae02afd6c27252f67370
[2023-05-18T07:21:14.006+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:14.009+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:14.102+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:14.103+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:14.172+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T07:21:14.201+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T07:21:14.202+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T07:21:14.243+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-18T07:21:14.260+0000] {standard_task_runner.py:57} INFO - Started process 1998 to run task
[2023-05-18T07:21:14.265+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmptk5i0hcy']
[2023-05-18T07:21:14.269+0000] {standard_task_runner.py:85} INFO - Job 43: Subtask clean_fondue_data
[2023-05-18T07:21:14.403+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:14.404+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:14.414+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 0a50b96f7e57
[2023-05-18T07:21:14.704+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-18T07:21:14.980+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T07:21:15.199+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T07:21:15.227+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230518T072114, end_date=20230518T072115
[2023-05-18T07:21:15.282+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T07:21:15.348+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-18T07:38:30.278+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:30.288+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:30.363+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:30.364+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:30.428+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T07:38:30.449+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [queued]>
[2023-05-18T07:38:30.450+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T07:38:30.478+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-12 00:00:00+00:00
[2023-05-18T07:38:30.488+0000] {standard_task_runner.py:57} INFO - Started process 965 to run task
[2023-05-18T07:38:30.492+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-12T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpgtgav6nr']
[2023-05-18T07:38:30.494+0000] {standard_task_runner.py:85} INFO - Job 32: Subtask clean_fondue_data
[2023-05-18T07:38:30.568+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:30.569+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-12T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:30.575+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-12T00:00:00+00:00 [running]> on host 4fe746fadf66
[2023-05-18T07:38:30.749+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-12T00:00:00+00:00'
[2023-05-18T07:38:30.886+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T07:38:31.001+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T07:38:31.024+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230512T000000, start_date=20230518T073830, end_date=20230518T073831
[2023-05-18T07:38:31.107+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T07:38:31.371+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
